explore-analyze/alerts-cases/alerts/query-alerts.md

```diff
@@ -0,0 +1,189 @@
+---
+navigation_title: How to query alert indices
+mapped_pages:
+  - https://www.elastic.co/guide/en/kibana/current/query-alerts.html
+applies_to:
+  stack: ga
+  serverless: ga
+products:
+  - id: kibana
+---
+
+# How to query alert indices [view-alerts]
+
+## Index Names
+
+On **Serverless** alerts are stored in [datastreams](https://www.elastic.co/docs/manage-data/data-store/data-streams), on on-prem and Elastic Cloud Hosted (ECH) they are stored in the indices.
+
+All the alert index names consist of 5 parts:
+
+All of them start with `.internal.alerts-` prefix.
+Then the `context`, `dataset`, `space-Id` and `version number` parts follow it.
+
+An index name template:<br>
+`.internal.alerts-{{context}}.{{dataset}}-{{space-id}}-{{version-number}}`
+
+<blockquote>
```

Comment:
Don't think this syntax is supported in our system. You may just need to add 3 spaces before the list items below. 

```suggestion
```